### PR TITLE
Win2 30 feature/share link

### PR DESCRIPTION
**Pull request by silksil**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -89,8 +89,9 @@
     "cra-webpack-rewired": "1.0.1",
     "crypto-browserify": "^3.12.0",
     "currency-codes": "^2.1.0",
-    "date-fns": "^2.29.3",
-    "dayjs": "^1.11.6",
+    "date-fns": "^2.29.1",
```
Comment (bobetbat) [inline package.json:92]: why downgraded?

File: src/utils/getFontValue.ts [hunk 2]
```diff
@@ -78,6 +78,7 @@ function useWidth() {
 
   return (
     keys.reduce((output, key) => {
+      // eslint-disable-next-line react-hooks/rules-of-hooks
       const matches = useResponsive('up', key);
```
Comment (bobetbat) [inline src/utils/getFontValue.ts:82]: hooks must be called in root of functions.(Fix needed out of scope)

File: src/utils/events.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { DateTime } from 'luxon';
+import { CoordinatesType } from './accommodation';
```
Comment (bobetbat) [inline src/utils/events.ts:2]: `import type` and it should go before regular imports

File: src/containers/search/SearchCard.tsx [hunk 4]
```diff
@@ -117,98 +122,115 @@ export const SearchCard = forwardRef<HTMLDivElement, SearchCardProps>(
...
           </Stack>
-          <Stack
-            // fill={true}
-            onClick={() => navigate(`/facility/${facility.id}`)}
-            justifyContent="space-between"
-            width="100%"
-            // minWidth={theme.spacing(27)}
-            spacing={0.5}
-            sx={{ p: 1, mt: 0, cursor: 'pointer' }}
+          <Link
```
Comment (dave-ok) [inline src/containers/search/SearchCard.tsx:125]: This opens in a new tab right?

File: src/hooks/useAccommodationsAndOffers/index.ts [hunk 5]
```diff
@@ -168,8 +165,10 @@ export const useAccommodationsAndOffers = ({
 
   // all normalized offers prior to filtering
   const allOffers = useMemo(
-    () => data?.offers && normalizeOffers(data.offers),
-    [data, preferredCurrencyCode]
+    () => data?.offers && normalizeOffers(data.offers, data.accommodations),
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (dave-ok) [inline src/hooks/useAccommodationsAndOffers/index.ts:170]: Why isn't ESLint happy with this?

File: src/hooks/useAccommodationsAndOffers/useAccommodationsAndOffersHelpers.ts [hunk 6]
```diff
@@ -25,28 +26,37 @@ export const useAccommodationsAndOffersHelpers = () => {
...
         ...value
       }));
 
-      let normalizedOffers: OfferRecord[];
+      let offersWithPreferredCurrency: OfferRecord[];
 
       if (preferredCurrencyCode) {
-        normalizedOffers = getOffersWithPreferredCurrency(
-          normalizedData,
+        offersWithPreferredCurrency = getOffersWithPreferredCurrency(
```
Comment (dave-ok) [inline src/hooks/useAccommodationsAndOffers/useAccommodationsAndOffersHelpers.ts:44]: The offers may not always include the preferred currency and as such, naming this way could be misleading

File: src/utils/getFontValue.ts [hunk 7]
```diff
@@ -78,6 +78,7 @@ function useWidth() {
 
   return (
     keys.reduce((output, key) => {
+      // eslint-disable-next-line react-hooks/rules-of-hooks
```
Comment (dave-ok) [inline src/utils/getFontValue.ts:81]: Is something not properly used here? Might need a refactor

**Review by dave-ok — COMMENTED** [review]
A couple of ESLint warnings popping up. I hope it doesn't break deployment

File: src/containers/search/SearchCard.tsx (hunk 4 above)
Comment (silksil) [reply src/containers/search/SearchCard.tsx:125]: yes, but  i will change this in a coming pr. Right now (also on test) a refresh happens if you go back from facility detail to search page. This is a short-term workaround to create a better experience

File: src/store/types.ts [hunk 8]
```diff
@@ -18,11 +19,12 @@ export interface GenericStateRecord {
   [key: string]: unknown;
 }
 
-interface OfferPreferredCurrencyPrice {
+interface OfferFrontend {
```
[the remark this answers is not in the document]
Comment (silksil) [reply src/store/types.ts:22]: Yes, i struggled coming with a better name. Let me know if you know a better name

File: src/utils/getFontValue.ts (hunk 7 above)
Comment (silksil) [reply src/utils/getFontValue.ts:81]: it is the same code as from the MUI template, so I kinda trust it. Also haven't experienced any problems with it , so I suggest to keep it like this.

**Review by dave-ok — APPROVED** [review]

**Pull request merged**